tests/conftest.py

```diff
@@ -106,3 +108,21 @@ def teardown():
 
     request.addfinalizer(teardown)
     return app
+
+
+@pytest.fixture()
+def communities_creation(app):
+    """Create some example communities."""
+    with app.app_context():
```

Comment:
It's better to refactor the app fixture to use the pattern here: https://github.com/inveniosoftware/invenio-records-rest/blob/master/tests/conftest.py#L172-L173 Also the DB fixture will be useful: https://github.com/inveniosoftware/invenio-records-rest/blob/master/tests/conftest.py#L179-L190